app/models/submission_state_actions.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class SubmissionStateActions
```

Comment:
Not sure if this is "the ruby way", but since there are now files associated with the `submission` model, would it make sense to do 

```ruby
- models
--- submission
------ submission.rb
------ submission_stater_actions.rb
```
?

Reply:
I guess I'd probably wait on something like this but as things evolve we could certainly move to a structure like this. The thing is that with a structure like that it implies a nesting and modules that IMO are overkill. 🤷